src/components/SeparatorVirtualGrid.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+	<h2 class="grid-title" :style="{height: item.height + 'px'}">
+		{{ item.injected.month }}
+		<span>{{ item.injected.year }}</span>
+	</h2>
+</template>
+
+<script>
+export default {
+	name: 'SeparatorVirtualGrid',
+
+	inheritAttrs: false,
+
+	props: {
+		item: {
+			type: Object,
+			required: true,
+			injected: {
```

Comment:
`injected` needs to go away.

Reply:
We can't remove injected, it's part of the lib. Everything inside the item is edited by the lib, `injected` is set by the user of the lib, it can be anything and the lib won't touch it, in our case it's whether an image or a separator.
If you want to better understand check out the demo of the lib.

Reply:
Removed, i misunderstood the comment sorry